tools/importer-rest-api-specs/components/parser/dataworkarounds/workaround_redis_22407.go

```diff
@@ -9,7 +9,7 @@ type workaroundRedis22407 struct {
 
 func (w workaroundRedis22407) IsApplicable(apiDefinition *models.AzureApiDefinition) bool {
 	serviceMatches := apiDefinition.ServiceName == "Redis"
-	apiVersionMatches := apiDefinition.ApiVersion == "2021-06-01" || apiDefinition.ApiVersion == "2022-05-01" || apiDefinition.ApiVersion == "2022-06-01"
+	apiVersionMatches := apiDefinition.ApiVersion == "2021-06-01" || apiDefinition.ApiVersion == "2022-05-01" || apiDefinition.ApiVersion == "2022-06-01" || apiDefinition.ApiVersion == "2023-04-01"
```

Comment:
We can remove the older versions since they aren't in use anymore (not in the `resource-manager.hcl` config)
```suggestion
	apiVersionMatches :=  apiDefinition.ApiVersion == "2022-06-01" || apiDefinition.ApiVersion == "2023-04-01"
```